### ISSUE TITLE
NixOS module systemd service

### ISSUE THREAD

**Issue by patrl** [kmonad/kmonad#745]
I've been using the systemd service provided by the NixOS module for over a year now. I've had a persistent issue where, if I rebuild my system, any devices with associated kmonad services become completely unresponsive, fixable by either un- and re-plugging the device, or restarting the associated service. This is generally tolerable, but today I had to hard reboot my laptop because the keyboard became unresponsive after a rebuild.

Here's my configuration, but it's pretty standard. The issue is presumably in the [service definition](https://github.com/kmonad/kmonad/blob/master/nix/nixos-module.nix). 

``` nix
services.kmonad = {
    enable = true;
    keyboards.laptop-internal = {
      device = "/dev/input/by-path/platform-i8042-serio-0-event-kbd";
      config = (builtins.readFile ./x1carbon.kbd);
      defcfg = {
        enable = true;
        fallthrough = true;
        compose.key = null;
      };
    };
```

**Comment by StrictlyMonad** [comment]
Likely culprit:
https://github.com/kmonad/kmonad/blob/8440eafdb516a379b4350f3c29a0e919f88cee24/nix/nixos-module.nix#L129

**Comment by montchr** [comment]
This was fixed in a1aea16c7a3239c0a3d7522b09edb1d6988bae02. Looks like there was a typo when mentioning this issue in the commit message.
